src/coreclr/gc/gc.cpp

```diff
@@ -45395,8 +45395,13 @@ HRESULT GCHeap::Initialize()
         }
         else
         {
+#ifdef TARGET_RISCV64
```

Comment:
if this is expected for more platforms, might make sense to `#ifdef` a constant for the size in GB, instead of the full block with all the calculations to minimize duplication.

Reply:
yep, for example, this could be defined after the block of code that defines `INITIAL_ALLOC`.

Reply:
Thanks!